modules/audio_device/mac/audio_device_mac.cc

```diff
@@ -867,12 +864,9 @@ int32_t AudioDeviceMac::RecordingDeviceName(uint16_t index,
   }
 
   memset(name, 0, kAdmMaxDeviceNameSize);
-
-  if (guid != NULL) {
```

Comment:
can anything else call this function and pass in NULL for `RecordingDeviceName` ?

Reply:
`RecordingDeviceName` is supposed to populate `name` and `guid` of the device,
but `guid` was not implemented.

So I return the `AudioDeviceID` which is not technically `guid` but 
it works for identifying the device.